### DISCUSSION TITLE
Moving Directus v9 to another Server on Digital Ocean

### DISCUSSION THREAD

**Discussion by akshatahuja03** [directus/directus#5200] [Archive]
I want to move my current v9 Directus to another server, which all directories should I copy to the newer server?
PS. I am using one click installation from Digital Ocean, and I now want to move my content to another account of Digital Ocean altogether. I am using postgresql installed on the server as DB.

Thank you in advance for your help :)

**Comment by rijkvanzanten** [comment]
Only your uploads directory (assuming you're using locally stored files, and not an external service). Other than that, a simple database export/import will do the trick 👍🏻

**Reply by akshatahuja03** [reply]
Okay, once again Thank you @rijkvanzanten

**Reply by biapar** [reply]
Which are the steps before copy uploads and extensions folders and restore db? 
Do We need to launch Directus install command?

**Reply by rijkvanzanten** [reply]
There are none, you simply copy the file uploads (if you're using a local folder), restore the database, and start directus on the new server 👍🏻

**Reply by biapar** [reply]
After a new Directus install i the same named folder?

**Reply by rijkvanzanten** [reply]
I don't know what you mean

**Comment by akki-akshat** [comment]
Hi, I tried it, but its not working properly: 

@rijkvanzanten maybe its a DB issue, it was working fine but then to migrate the Postgres DB, I exported the dump and tried to apply the dump, it was giving an error so I dropped the directus database and then added the dump. There were no errors then. 

I am getting these errors in console:
<img width="1428" alt="image" src="https://user-images.githubusercontent.com/19926797/123667547-e0c9ad80-d857-11eb-99fe-bcee9b908d13.png">

IP address to access the directus server is: http://165.22.221.177/admin/

**Reply by akki-akshat** [reply]
Also, to be able to authenticate Postgres, I changed auth mode to md5 from peer and changed it back. I did restart the Postgres after it.

**Reply by akki-akshat** [reply]
I have also provided the required privileges to directus on Postgres as it was there on a working instance of directus
![image](https://user-images.githubusercontent.com/19926797/123669582-df998000-d859-11eb-8e81-518abf08ce12.png)

**Reply by rijkvanzanten** [reply]
Not sure if this is a problem in the DB itself! Are there any logs on the server itself (where you're running the Directus process)

**Reply by akki-akshat** [reply]
Logs from PM2 for db is as follows: 
![Screenshot 2021-06-28 at 11 36 57 PM](https://user-images.githubusercontent.com/19926797/123683490-e16b3f80-d869-11eb-86f9-a452b665594b.png)

**Reply by rijkvanzanten** [reply]
Are you using a newer version on Directus on the old database? Try running `npx directus database migrate:latest` and restart the server

**Comment by akki-akshat** [comment]
I found the issue, I am yet to test it though, I have copied the data from the version 9.0.0-rc.51 to 9.0.0-rc81 as I am using one click deploy from digital ocean.

There have been changes in the schema of directus_collection
New one:
![Screenshot 2021-06-28 at 11 59 43 PM](https://user-images.githubusercontent.com/19926797/123686275-44aaa100-d86d-11eb-9c68-72aa4726b331.png)

Old one
![image](https://user-images.githubusercontent.com/19926797/123686529-850a1f00-d86d-11eb-9405-72eb0e991ffa.png)

Let me know @rijkvanzanten if I am on the right way

**Reply by rijkvanzanten** [reply]
That's exactly what I thought and suggested in my last reply 🙂 

> Are you using a newer version on Directus on the old database? Try running npx directus database migrate:latest and restart the server

**Reply by akki-akshat** [reply]
That worked, Thank you :)
